lib/trino-cache/src/test/java/io/trino/cache/TestEvictableCache.java

```diff
@@ -585,4 +588,26 @@ public void testPutOnNonEmptyCacheImplementation()
                 .isInstanceOf(UnsupportedOperationException.class)
                 .hasMessage("The operation is not supported, as in inherently races with cache invalidation");
     }
+
+    @RepeatedTest(1_000)
+    public void testParallelLoadingCacheEntries()
+    {
+        Cache<String, String> cache = EvictableCacheBuilder.newBuilder()
+                .expireAfterWrite(Duration.ofSeconds(60))
+                .maximumSize(10)
+                .build();
+        try (ExecutorService executor = Executors.newFixedThreadPool(2)) {
+            Runnable longRunningCacheLoad = () -> {
+                try {
+                    cache.get("key", () -> "value");
+                }
+                catch (ExecutionException e) {
+                    throw new RuntimeException(e);
+                }
+            };
```

Comment:
What makes it "long running cache load"?

Reply:
its name I used previously when I had `Thread.sleep` inside cache loader to simulate long cache loading, but I swapped out `Thread.sleep` for `RepeatedTest` sake, will rename